x/mint/keeper/keeper.go

```diff
@@ -181,7 +179,7 @@ func (k Keeper) AddCollectedFees(ctx sdk.Context, fees sdk.Coins) error {
 	err := k.bankKeeper.SendCoinsFromModuleToModule(ctx, types.ModuleName, k.feeCollectorName, fees)
 
 	// allocate pool allocation ratio to pool-incentives module account account
-	coins := sdk.NewCoins(k.GetPoolAllocatableAsset(ctx))
+	coins := sdk.NewCoins(k.GetPoolAllocatableAsset(ctx, fees))
 	err = k.bankKeeper.SendCoinsFromModuleToModule(ctx, k.feeCollectorName, poolincentivestypes.ModuleName, coins)
```

Comment:
These coins shouldn't go to the fee collector first.  This function should send the proper amount to fee collector bucket, and poolincentives type.